JSDemos/Demos/Scheduler/RecurringAppointments/description.md

```diff
@@ -1 +1,19 @@
-This example demonstrates how to&nbsp;define recurring appointments. You can specify the appointment repetition interval programmatically or&nbsp;in&nbsp;the UI. In&nbsp;the former case, the appointment object should contain the **recurrenceRule** field. In&nbsp;the latter case, toggle the **Repeat** switcher on&nbsp;the appointment details form to&nbsp;mark the appointment as&nbsp;recurring and specify the required fields.
\ No newline at end of file
+A recurring appointment repeats at a specified interval. You can create such an appointment in the UI or programmatically.
+
+To create a recurring appointment programmatically, do the following:
+
+1. Locate or create only a single appointment object in the [dataSource](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#dataSource). 
+
+2. Specify the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) where you should generate a recurrent series according to the <a href="http://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification.
```

Comment:
A framework or a component can generate something but I'm not sure that people can. Also, what is 'a recurrent series' in context of Scheduler?